pyproject.toml

```diff
@@ -42,7 +42,7 @@ classifiers = [
 python = "^3.7"
 brainstem = "*"
 PyYAML = "^6.0"
-pylink-square = ">=0.12,<0.15"
+pylink-square = ">=0.12,<1.1"
```

Comment:
I just see on the pylink-square changelog :

Changed .flash() to no longer erase chip on flash; users will now need to ensure they call .erase() prior to flashing a non-erased region of flash.

Does this behavior the usage of our flasher?